firestore/src/android/query_snapshot_android.h

```diff
@@ -65,6 +65,13 @@ class QuerySnapshotInternal : public Wrapper {
   std::size_t size() const;
 };
 
+bool operator==(const QuerySnapshotInternal& lhs,
+                const QuerySnapshotInternal& rhs);
+inline bool operator!=(const QuerySnapshotInternal& lhs,
```

Comment:
Nit: separate the functions by an empty line?

Reply:
Done